src/coreclr/nativeaot/Runtime/threadstore.cpp

```diff
@@ -230,7 +230,7 @@ void SpinWait(int iteration, int usecLimit)
     int64_t ticksPerSecond = PalQueryPerformanceFrequency();
     int64_t endTicks = startTicks + (usecLimit * ticksPerSecond) / 1000000;
 
-    int l = min((unsigned)iteration, 30);
+    int l = (int)min((unsigned)iteration, 30u);
```

Comment:
```suggestion
    int l = iteration >= 0 ? min(iteration, 30) : 30;
```
The converting of `-1` to a max unsigned is a lot of cleverness for one line.